### ISSUE TITLE
demoBebopVisioGUI.py problem

### ISSUE THREAD

**Issue by ZisisT** [amymcgovern/pyparrot#167]
so I'm trying to run this script and I get this error:

[00007f96e6b71070] videotoolbox decoder: Using Video Toolbox to decode 'h264'
[00007f96e6b71070] main decoder error: buffer deadlock prevented
libpng error: Read Error
[00007f96e6b71070] main decoder error: buffer deadlock prevented
[00007f96e6b71070] videotoolbox decoder: Using Video Toolbox to decode 'h264'
[00007f96e6b71070] videotoolbox decoder: vt cvpx chroma: 420v

Basically from what I can tell the VLC video playback is not working, the image is being saved tho. Any ideas as to what might this be? I've already updated libpng

**Comment by amymcgovern** [comment]
it works fine.  It just puts out a lot of information from VLC.  the ffmpeg version does the same thing.  Lots of output!  But the images are fine.

**Comment by kent9jp** [comment]
I have this similar error in demoAnafiVisionGUI.py


Starting user vision thread
user callback being called
success from play call is 0 
Created new TCP socket 35 for connection
timeout - trying again
[00007fc67804cb70] main decoder error: buffer deadlock prevented
[00007fc67804cb70] avcodec decoder: Using NVIDIA VDPAU Driver Shared Library  460.91.03  Fri Jul  2 05:42:23 UTC 2021 for hardware decoding
[00007fc6bc000be0] main video output error: Failed to grab a snapshot
timeout - trying again
timeout - trying again
timeout - trying again
...


I was able to stream the video with the GUI only once, but after that, a black screen or the visionStream.jpg loaded on the first time is displayed.
Is there any special installation process for vlc or ffmpeg?
Also, demoAnafiVision.py seems to work fine and a lot of pngs are stored in ./pyparrot/images.

env
OS:ubunts18.04    python==3.6.9
